Sources/Classes/SMStoreChangeSetHandler.swift

```diff
@@ -124,6 +124,18 @@ class SMStoreChangeSetHandler {
         changeSet.setValue(entityName, forKey: SMStoreChangeSetHandler.SMLocalStoreEntityNameAttributeName)
         changeSet.setValue(NSNumber(value: SMLocalStoreRecordChangeType.recordInserted.rawValue as Int16), forKey: SMStoreChangeSetHandler.SMLocalStoreChangeTypeAttributeName)
     }
+  
+  func countOfChangeSet(backingContext: NSManagedObjectContext) -> Int {
+    let r = NSFetchRequest<NSFetchRequestResult>(entityName: SMStore.SMLocalStoreChangeSetEntityName)
+    do {
+      let c =   try backingContext.count(for: r)
+      VERBOSE("OK There are \(c) items ready for upload")
+      return c
+    } catch {
+      ERROR("ERROR fetching from \(SMStore.SMLocalStoreChangeSetEntityName): \(error)")
+      return -1
```

Comment:
Functions should not return -1 to indicate an error; it would be more appropriate to `throw`

Reply:
Will change